Jenkinsfile

```diff
@@ -186,7 +186,7 @@ stage('Build') {
           sh "${docker_run} ${ci_cpu} ./tests/scripts/task_python_integration.sh"
           sh "${docker_run} ${ci_cpu} ./tests/scripts/task_python_vta_fsim.sh"
           sh "${docker_run} ${ci_cpu} ./tests/scripts/task_python_vta_tsim.sh"
-          // sh "${docker_run} ${ci_cpu} ./tests/scripts/task_golang.sh"
+          sh "${docker_run} ${ci_cpu} ./tests/scripts/task_golang.sh"
```

Comment:
please first temporary disable this line, since the script won't take effect until we get it to main. Let us first merge the rest. Then stage the go tests in case  something go wrong

Reply:
done